src/random_search_eda.py

```diff
@@ -0,0 +1,43 @@
+# author: Arushi Ahuja
+# date: 2021-11-24
+
+'''This script generates all the EDA diagrams used in the final report 
+  after model generation, specifically the random search diagram
+
```

Comment:
The description is the same as `credit default eda.py` . 😄

Reply:
No I changed it to "after model generation"